src/cheri/xycheriot.adoc

```diff
@@ -0,0 +1,678 @@
+grant exactly all of <<w_perm>>, <<c_perm>>, and <<zylevels1_sl_perm>>.
+These capabilities were judged to be the least useful of the combinations
+left after the by-construction properties of the roots and the myriad
+<<CLRPERM>> rules had winnowed the possible combinations.
+It is easy to verify that all other permission bits are determined by
+the presence of <<zylevels1_sl_perm>> and the absence of <<r_perm>>:
+
+* The presence of <<zylevels1_sl_perm>> already, by its definition,
+  requires the presence of <<w_perm>> and <<c_perm>>.
+
+* The presence of <<w_perm>> implies both
+
+** that neither <<x_perm>> nor <<asr_perm>> are granted
+   (the first "by construction" property and
+   <<asr_perm>>'s dependence on <<x_perm>>) and
+
+** that none of
+   <<zyseal_se_perm>>, <<zyseal_us_perm>>, or the `U0` permission
+   are granted (the second "by construction" property).
+
+* The absence of <<r_perm>> similarly implies absence of
+  its dependents, <<lm_perm>> and <<zylevels1_lg_perm>>.
+
+The only permission bit left to consider is <<zylevels1_gl_perm>>.
+Thus, making <<zylevels1_sl_perm>> depend on <<r_perm>> has the effect of
+precluding exactly the two stated capability forms.
+=====
+
+[NOTE]
+=====
+TODO: the actual encoding format is given
+in §7.13.1 / "Capability permissions" of {cheriot_v1_isa}.
+=====
+
+==== <<sec_cap_type>> Values and Encoding
+
+{cheriot_ext_name} uses a 3-bit field to encode <<sec_cap_type>> values.
+Unsealed capabilities, those with type `0`, have an all-zeros type field,
+and an all-zeros type field always corresponds to an unsealed capability.
+Otherwise, the interpretation of this field
+_depends on the capability's <<x_perm>> value_:
+field values `0b001` through `0b111` correspond to types...
+
+* {cheriot_ot_x_min} (inclusive) through {cheriot_ot_x_max} (inclusive)
+  iff the capability grants <<x_perm>>, or
+
+* {cheriot_ot_nx_min} (inclusive) through {cheriot_ot_nx_max} (inclusive)
+  iff the capability does not grant <<x_perm>>.
+
+That is, it is not possible to express a capability with type between
+{cheriot_ot_x_min} (inclusive) through {cheriot_ot_x_max} (inclusive)
+that does not grant <<x_perm>> or, dually, one with type between
+{cheriot_ot_nx_min} (inclusive) through {cheriot_ot_nx_max} (inclusive)
+that does grant <<x_perm>>.
+Attempts to <<YSEAL>> or otherwise construct such a capability,
+or one with type equal to {cheriot_ot_absent} or above {cheriot_ot_max},
+will instead result in a cleared {ctag}.
+
+_No_ non-zero <<sec_cap_type>> value is considered
+<<sec_cap_type_ambient,ambiently available>>,
+even the <<sentry_cap>> types (see below).
+
+[NOTE]
+=====
+The lack of ambient types has implications:
+
+* The <<section_rvy_sentry_insn_ext>> extension (and its <<SENTRY>> instruction)
+  requires an ambiently available type,
+  and so <<section_rvy_sentry_insn_ext>> is incompatible with {cheriot_ext_name}.
+
+* If the result of a <<CBLD>> instruction has a set {ctag},
```

Comment:
This is a weird way of stating it. The easier way to parse is if the input is sealed, then the output will be untagged.

Reply:
After #767, the `YBLD` semantics are https://github.com/riscv/riscv-cheri/blob/c93971925926714489aa14946adba9ca2d75e104/src/cheri/insns/cbld_32bit.adoc?plain=1#L39-L43 .  That is, `YBLD` will prefer to build a tagged and unsealed thing rather than an untagged thing if the input bit patterns call for a non-ambient sealing type.